src/pypromice/postprocess/csv2bufr.py

```diff
@@ -187,46 +187,34 @@ def setAWSvariables(
     setBUFRvalue(ibufr, 'hour', timestamp.hour)
     setBUFRvalue(ibufr, 'minute', timestamp.minute)
 
-    vars_dict = {
-        'relativeHumidity': 'rh_i', # DMI wants non-corrected rh
-        'airTemperature': 't_i',
-        'pressure': 'p_i',
-        'windDirection': 'wdir_i',
-        'windSpeed': 'wspd_i'
-    }
-    for bufr_key, source_var in vars_dict.items():
-        if (stid in vars_to_skip) and (source_var in vars_to_skip[stid]):
-            print('----> Skipping var: {} {}'.format(stid,source_var))
-        else:
-            setBUFRvalue(ibufr, bufr_key, row[source_var])
+    setBUFRvalue(ibufr, "relativeHumidity", row["relativeHumidity"])
+    setBUFRvalue(ibufr, "airTemperature", row["airTemperature"])
+    setBUFRvalue(ibufr, "pressure", row["pressure"])
+    setBUFRvalue(ibufr, "windDirection", row["windDirection"])
+    setBUFRvalue(ibufr, "windSpeed", row["windSpeed"])
+
+    setBUFRvalue(ibufr, 'latitude', row['latitude'])
 
     # Set position metadata
-    setBUFRvalue(ibufr, 'latitude', row['gps_lat_fit'])
```

Comment:
Not sure I follow why you are no longer using the smoothed ("fit") version of lat and lon. It is important to use smoothed positions. But probably I just don't follow your changes here...

Reply:
It is still using the smoothed version when the variables are selected and calculated in `get_bufr.get_bufr_variables`. I admit that the structure, naming and docstrings are nok clear enough.

https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/features/get_bufr_restructure/src/pypromice/postprocess/get_bufr.py#L291